src/components/elements/upload/fileSummary/FileUploadSummary.tsx

```diff
@@ -0,0 +1,38 @@
+import { Typography } from '@mui/material';
+import React, { useMemo } from 'react';
+import FileSummary, {
+  FileSummaryProps,
+} from '@/components/elements/upload/fileSummary/FileSummary';
+import theme from '@/config/theme';
+
+// Wrapper around FileSummary for representing a File, aka a file that has just now been uploaded to Active Storage but doesn't yet exist as a File record in our db.
+const FileUploadSummary: React.FC<{
+  file: File;
+  variant: FileSummaryProps['variant'];
+  onRemove?: FileSummaryProps['onRemove'];
+}> = ({ file, variant, onRemove }) => {
+  const url = useMemo(() => URL.createObjectURL(file), [file]);
+
+  return (
+    <>
+      <FileSummary
+        fileName={file.name}
+        showThumbnail={!!file.type.match(/^image/)}
+        url={url}
+        info={
+          <Typography
+            variant='body2'
+            sx={{ color: theme.palette.warning.main }}
```

Comment:
This contrast is not accessible, "dark" is a little better, but still flagged by storybook. 
Linking https://github.com/open-path/Green-River/issues/7108 which will introduce a warning text color, which _will_ be accessible. Maybe add  a todo?

```suggestion
            color='warning.dark' # TODO(#718) use warning text color
```